File: Liguagem C#/1011.cs
using System; 

class URI {

    static void Main(string[] args) { 
    
    int raio = int.Parse(Console.ReadLine());
    double pi = 3.14159;
    double volumeEsfera = (4.0/3) * pi * Math.Pow(raio,3.0);
    Console.WriteLine("VOLUME = {0:F3}",volumeEsfera);
 
    }
}

File: Linguagem Dart/1011.dart
import 'dart:io';

void main()
{
    int raio = int.parse(stdin.readLineSync());
    
    double volumeEsfera = (4.0/3.0) * 3.14159 * (raio*raio*raio);
    print("VOLUME = " + volumeEsfera.toStringAsFixed(3));
}
